ga4gh/frontend.py

```diff
@@ -169,6 +173,7 @@ def searchVariantSets():
     return handleFlaskRequest(flask.request, app.backend.searchVariantSets)
 
 
-@app.route('/variants/search', methods=['POST', 'OPTIONS'])
+@app.route('/<version>/variants/search', methods=['POST', 'OPTIONS'])
+@versioningManager.policy('V_0.5.1_RELEASE')
```

Comment:
Really tiny nit: if we're using VX.Y.Z for version strings, why add the extra underbar to make V_X.Y.Z here?